CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
         * `continuation_prompt`
         * `self_in_py`
         * `prompt`
+    * `self.settable` changed to `self.settables`
```

Comment:
We should probably add something under `Enhancements` about the `add_settable()` and `remove_settable()` methods